scripts/tests/chiptest/accessories.py

```diff
@@ -116,3 +118,13 @@ def __startXMLRPCServer(self):
 
     def __stopXMLRPCServer(self):
         self.server.shutdown()
+
+    def __createCommandLineOptions(self, args):
+        # args should contain a list of strings in the form of [option1, value1, option2, value2, ...]
```

Comment:
Why not a list of tuples like [(option1, value1), (option2, value2)]?  Or better yet just the dictionary form to start with?  Either would be less error-prone at the caller than this, I think.

Reply:
Updated comments in both `start` and `reboot` in this file to clearly specify the args param